### PR TITLE
Add headers to santa maria legacy gateway

### PR DESCRIPTION
**Pull request by danielburnley**
**WHAT** - Adds the headers to the santa maria legacy gateway requests

**WHY** - in order to make requests to SM in legacy mode, the request 
needs certain headers which were not implemented

### CODE REVIEW

File: lib/santa_maria/gateway/santa_maria_legacy.rb [hunk 1]
```diff
@@ -48,7 +46,18 @@ def variants_for(global_id)
 
       private
 
-      attr_reader :endpoint
+      attr_reader :endpoint, :site_code, :language
+
+      def get(uri)
+        request = Net::HTTP::Get.new(uri)
```
Comment (danielburnley) [inline lib/santa_maria/gateway/santa_maria_legacy.rb]: Net::HTTP is so grim

Comment (naliwajek) [reply lib/santa_maria/gateway/santa_maria_legacy.rb]: Ha, yeah, we did spend some time with Kyle trying to make it work for v2 gateway ;fff

Comment (naliwajek) [reply lib/santa_maria/gateway/santa_maria_legacy.rb]: But that's a gem – better to keep deps low.

Comment (danielburnley) [reply lib/santa_maria/gateway/santa_maria_legacy.rb]: :+1: agreed

**Pull request merged**